### PR TITLE
Hw1

### PR DESCRIPTION
**Pull request by oleksii-devico**

### CODE REVIEW

File: cypress/e2e/sign_in_page.cy.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/// <reference types="cypress" />
+
```
Comment (ievgendco) [inline cypress/e2e/sign_in_page.cy.js:2]: Here should no be a space

File: cypress/e2e/sign_in_page.cy.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import { sign_in_page } from "../selectors/sign_in_page";
+
+describe('UI tests for sign in page', () => {
+
+  before('visiting sign in page', () => {
+    cy.visit('/')
+  })
+  it('should show "Real World App logo"', () => {
```
Comment (ievgendco) [inline cypress/e2e/sign_in_page.cy.js:10]: Single space should be after hook and after each test (it) like this:
before() {
//something
}

it('test1'), {
//something
}

it('test2'), {
//something
}

**Review by ievgendco — CHANGES_REQUESTED** [review]

File: cypress/e2e/sign_in_page.cy.js (hunk 1 above)
Comment (oleksii-devico) [reply cypress/e2e/sign_in_page.cy.js]: resolved!

File: cypress/e2e/sign_in_page.cy.js (hunk 2 above)
Comment (oleksii-devico) [reply cypress/e2e/sign_in_page.cy.js:10]: Resolved!

**Pull request merged**